cmd/main.go

```diff
@@ -86,7 +92,16 @@ func main() {
 		"Enable metrics collection for all Policy Servers and the Kubewarden Controller")
 	flag.BoolVar(&enableTracing, "enable-tracing", false,
 		"Enable tracing collection for all Policy Servers")
+	flag.BoolVar(&enableOtelSidecar, "enable-otel-sidecar", false,
+		"Enable OpenTelemetry sidecar in Policy Servers")
+
 	flag.StringVar(&openTelemetryEndpoint, "opentelemetry-endpoint", "127.0.0.1:4317", "The OpenTelemetry connection endpoint")
+	flag.StringVar(&openTelemetryCertificate, "opentelemetry-certificate", "", "")
+	flag.StringVar(&openTelemetryClientCertificateKey, "opentelemetry-client-certificate-key", "", "")
+	flag.StringVar(&openTelemetryClientCertificate, "opentelemetry-client-certificate", "", "")
+	flag.StringVar(&openTelemetryClientCertificateSecret, "opentelemetry-client-certificate-secret", "", "")
+	flag.StringVar(&openTelemetryCertificateSecret, "opentelemetry-certificate-secret", "", "")
```

Comment:
I would like to try to simplify all these CLI flags.

Reply:
On second thought, I think we can remove those if we are using the `OTEL_*` env variables.